### ISSUE TITLE
Graphics crash on initial boot

### ISSUE THREAD

**Issue by CaptainThrowback** [chubbzlou01/Sprint_A9_port#1]
Let device boot up, and when welcome screen appears, do nothing. TalkBack should begin speaking, and before completed, the device should restart the boot animation.

https://www.dropbox.com/s/4bonvjis9dgy7wk/logcat_A9_port.txt?dl=0
